Source/Eto.WinForms/Forms/Controls/DrawableHandler.cs

```diff
@@ -29,6 +29,27 @@ public EtoDrawable ()
 				this.SetStyle (swf.ControlStyles.SupportsTransparentBackColor, true);
 			}
 
+			public override sd.Rectangle DisplayRectangle
+			{
+				get
+				{
+					var rect = ClientRectangle;//base.DisplayRectangle;
+					var padding = Padding;
+					rect.X += padding.Left;
+					rect.Y += padding.Top;
+					rect.Width -= padding.Horizontal;
+					rect.Height -= padding.Vertical;
+					return rect;
+				}
+			}
+
+			public override sd.Size GetPreferredSize(sd.Size proposedSize)
+			{
+				if (Controls.Count > 0)
+					return Controls[0].GetPreferredSize(proposedSize - Padding.Size) + Padding.Size;
+				return base.GetPreferredSize(proposedSize);
+			}
+
```

Comment:
There is alwasy atmost one control - the content. But `swf.Control` does not handle the logic, `swf.Panel` would.
`return Padding.Size` as the last line could be better.